TyPython/src/CPython.ORM.jl

```diff
@@ -210,7 +210,7 @@ end
     result = Expr(:tuple, [:($py_coerce($t, $unsafe_pytuple_get(py, $(i - 1)))) for (i, t) in enumerate(T.parameters)]...)
     quote
         if $CPython.PyAPI.PyObject_IsInstance(py,  $CPython.PyAPI.PyTuple_Type) == 0
-            $CPython.PyAPI.PyErr_SetString($CPython.PyAPI.PyExc_TypeError[], "expected a tuple")
+            $py_seterror!(G_PyBuiltin.TypeError, "expected a tuple")
```

Comment:
~~We shall not use `G_PyBuiltin` here.~~

~~Probably `$get_py_builtin().TypeError`.~~

Reply:
P.S: It's okay to use `G_PyBuiltin` here as it is inside the AST returned by the generated function. 

This means the scope to resolve symbols is always the module that the generated function is defined in.